src/test/java/jp/ikedam/jenkins/plugins/updatesitesmanager/DescribedUpdateSiteSimpleTest.java

```diff
@@ -24,204 +24,90 @@
-            );
-        }
+
+    @Test
+    public void testDescriptorDoCheckIdErrorEmpty() {
+        assertThat("empty", getDescriptor().doCheckId("").kind, is(ERROR));
     }
-    
-    public void testDescriptorDoCheckIdError()
-    {
-        DescribedUpdateSite.Descriptor descriptor = getDescriptor();
-        
-        // null
-        {
-            String id = null;
-            assertEquals(
-                "null",
-                FormValidation.Kind.ERROR,
-                descriptor.doCheckId(id).kind
-            );
-        }
-        
-        // empty
-        {
-            String id = "";
-            assertEquals(
-                "empty",
-                FormValidation.Kind.ERROR,
-                descriptor.doCheckId(id).kind
-            );
-        }
-        
-        // blank
-        {
-            String id = "  ";
-            assertEquals(
-                "blank",
-                FormValidation.Kind.ERROR,
-                descriptor.doCheckId(id).kind
-            );
-        }
+
+    @Test
+    public void testDescriptorDoCheckIdErrorBlank() {
+        assertThat("blank", getDescriptor().doCheckId("  ").kind, is(ERROR));
     }
-    
-    public void testDescriptorDoCheckUrlOk()
-    {
-        DescribedUpdateSite.Descriptor descriptor = getDescriptor();
-        
-        {
-            String url = "http://localhost/update-center.json";
-            assertEquals(
-                "simple url",
-                FormValidation.Kind.OK,
-                descriptor.doCheckUrl(url).kind
-            );
-        }
+
+    @Test
+    public void testDescriptorDoCheckUrlOk() {
+        String url = "http://localhost/update-center.json";
+        assertThat("simple url", getDescriptor().doCheckUrl(url).kind, is(OK));
     }
-    
-    public void testDescriptorDoCheckUrlError()
-    {
-        DescribedUpdateSite.Descriptor descriptor = getDescriptor();
-        
-        // null
-        {
-            String url = null;
-            assertEquals(
-                "null",
-                FormValidation.Kind.ERROR,
-                descriptor.doCheckUrl(url).kind
-            );
-        }
-        
-        // empty
-        {
-            String url = "";
-            assertEquals(
-                "empty",
-                FormValidation.Kind.ERROR,
-                descriptor.doCheckUrl(url).kind
-            );
-        }
-        
-        // blank
-        {
-            String url = "  ";
-            assertEquals(
-                "blank",
-                FormValidation.Kind.ERROR,
-                descriptor.doCheckUrl(url).kind
-            );
-        }
-        
-        // non url
-        {
-            String url = "hogehoge";
-            assertEquals(
-                "non url",
-                FormValidation.Kind.ERROR,
-                descriptor.doCheckUrl(url).kind
-            );
-        }
+
+    @Test
+    public void testDescriptorDoCheckUrlError() {

```

Comment:
Don't be lazy here, please.
You refactored tests as you thought the existing ones were bad, did you?
(And I agree with that. I didn't know much about tests and I now believe your new ones are desirable)
Someone should refactor it again in future.